shared/common-adapters/icon.native.js

```diff
@@ -5,9 +5,7 @@ import ClickableBox from './clickable-box'
 import * as React from 'react'
 import {globalColors, glamorous} from '../styles'
 import {iconMeta} from './icon.constants'
```

Comment:
does this work at a?? what happens if i set color/hovercolor/fontSize etc?

Reply:
ah I see, color and fontSize weren't hooked up after being pulled out of style. But hoverColor is ignored on native isn't it?

Reply:
Yes but the others work